app/services/automation_rules/conditions_filter_service.rb

```diff
@@ -1,10 +1,10 @@
 require 'json'
 
 class AutomationRules::ConditionsFilterService < FilterService
-  def initialize(rule, conversation)
+  def initialize(rule, conversation_ids = nil)
```

Comment:
`conversation_ids` could be confusing as it gives an impression that we are passing an array of `conversation_ids`. But from my understanding, we are only passing a single conversation_id?